[book] mention suiup in install guid (#149)
MystenLabs/move-book: book/before-we-begin/install-sui.md

programs. The compiler is included into the Sui binary, which can be installed or downloaded using
one of the methods below.

## Installing via suiup

> The best way to install Sui is by using [`suiup`](https://github.com/MystenLabs/suiup). Not only
> does it provide an easy way to install a binary, but also manage different versions of binaries for
> different environments (e.g. `testnet` and `mainnet`).

Reviewer: Just a nit: ```suggestion
The best way to install Sui is by using [`suiup`](https://github.com/MystenLabs/suiup). It provides a simple way to install binaries and to manage different versions of binaries for
different environments (e.g. `testnet` and `mainnet`).
```
